node/src/BaseClient.ts

```diff
@@ -2367,6 +2368,23 @@ export class BaseClient {
         return this.createWritePromise(createPfAdd(key, elements));
     }
 
+    /** Returns the internal encoding for the Redis object stored at `key`.
+     *
+     * See https://valkey.io/commands/object-encoding for more details.
+     *
+     * @param key - The `key` of the object to get the internal encoding of.
+     * @returns - If `key` exists, returns the internal encoding of the object stored at `key` as a string.
+     *     Otherwise, returns None.
+     * @example
+     * ```typescript
+     * const result = await client.object_encoding("my_hash");
+     * console.log(result); // Output: "listpack"
+     * ```
+     */
+    public object_encoding(key: string): Promise<string | null> {
```

Comment:
According to the document, this command will return either nil or bulk string. I just want to double confirm that I have handled it correctly here.

Reply:
Yes, RESP bulk string is just a string: https://github.com/Bit-Quill/glide-for-redis/blob/eac0384c447b2f02404bfc1e4e95ee3f378dadd6/node/rust-client/src/lib.rs#L167-L170